app/hotels/src/allHotels/searchForm/locationPicker/RecentSearchItem.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from 'react';
+import { Touchable, Text } from '@kiwicom/react-native-app-shared';
+import Translations from '@kiwicom/react-native-app-translations';
+
+import type { Location } from './Types';
+import styles from './Styles';
```

Comment:
@tbergq Could you please make it more reusable? I would prefer to have the style encapsulated into component instead of requiring and using this style directly. One reason for this is a question from yesterday: "Could we reuse the suggestions picker for something else? For example flight locations?".

So instead of `Touchable` with required style let's introduce `Row` component related to our picker. What do you think?

Reply:
yes makes sense